src/style.css

```
@@ -0,0 +1,28 @@
+.page {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: 0 50%;
+  height: 16px;
+  width: 16px;
+  margin: 0 4px 0 2px;
+}
+
+.stars--1 > .stars__star:nth-child(-n+1),
+
+.stars--2 > .stars__star:nth-child(-n+2),
+
+.stars--3 > .stars__star:nth-child(-n+3),
+
+.stars--4 > .stars__star:nth-child(-n+4),
+
+.stars--5 > .stars__star:nth-child(n) {
```

Не залишай пусті лінійки між селекторами
```suggestion
.stars--1 > .stars__star:nth-child(-n+1),
.stars--2 > .stars__star:nth-child(-n+2),
.stars--3 > .stars__star:nth-child(-n+3),
.stars--4 > .stars__star:nth-child(-n+4),
.stars--5 > .stars__star:nth-child(n) {
```

---

то все лінтер( А з марджіном я не розумію шо за прикол)